<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audacious</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="aud_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFiles"/>
    <addaction name="actionOpenURL"/>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddURL"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionLogInspector"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionRepeat"/>
    <addaction name="actionShuffle"/>
    <addaction name="actionNoPlaylistAdvance"/>
    <addaction name="actionStopAfterThisSong"/>
    <addaction name="separator"/>
    <addaction name="actionSongInfo"/>
    <addaction name="actionJumpToTime"/>
    <addaction name="separator"/>
    <addaction name="actionSetRepeatPointA"/>
    <addaction name="actionSetRepeatPointB"/>
    <addaction name="actionClearRepeatPoints"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <widget class="QMenu" name="menuSort">
     <property name="title">
      <string>Sort</string>
     </property>
     <property name="icon">
      <iconset theme="view-sort-ascending"/>
     </property>
     <addaction name="actionSortTrackNumber"/>
     <addaction name="actionSortTitle"/>
     <addaction name="actionSortArtist"/>
     <addaction name="actionSortAlbum"/>
     <addaction name="actionSortDate"/>
     <addaction name="actionSortLength"/>
     <addaction name="actionSortPath"/>
     <addaction name="actionSortCustomTitle"/>
     <addaction name="separator"/>
     <addaction name="actionPlaylistReverse"/>
     <addaction name="actionPlaylistRandomize"/>
    </widget>
    <widget class="QMenu" name="menuSort_Selected">
     <property name="title">
      <string>Sort Selected</string>
     </property>
     <property name="icon">
      <iconset theme="view-sort-ascending"/>
     </property>
     <addaction name="actionSortSelectedTrackNumber"/>
     <addaction name="actionSortSelectedTitle"/>
     <addaction name="actionSortSelectedArtist"/>
     <addaction name="actionSortSelectedAlbum"/>
     <addaction name="actionSortSelectedDate"/>
     <addaction name="actionSortSelectedLength"/>
     <addaction name="actionSortSelectedPath"/>
     <addaction name="actionSortSelectedCustomTitle"/>
     <addaction name="separator"/>
     <addaction name="actionSelectedReverse"/>
     <addaction name="actionSelectedRandomize"/>
    </widget>
    <addaction name="actionPlayCurrentPlaylist"/>
    <addaction name="actionPlaylistRefresh"/>
    <addaction name="separator"/>
    <addaction name="menuSort"/>
    <addaction name="menuSort_Selected"/>
    <addaction name="actionPlaylistRemoveDuplicates"/>
    <addaction name="actionPlaylistRemoveUnavailableFiles"/>
    <addaction name="separator"/>
    <addaction name="actionPlaylistNew"/>
    <addaction name="actionPlaylistRename"/>
    <addaction name="actionPlaylistRemove"/>
    <addaction name="separator"/>
    <addaction name="actionPlaylistImport"/>
    <addaction name="actionPlaylistExport"/>
    <addaction name="separator"/>
    <addaction name="actionPlaylistManager"/>
    <addaction name="actionQueueManager"/>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>Output</string>
    </property>
    <addaction name="actionVolumeUp"/>
    <addaction name="actionVolumeDown"/>
    <addaction name="separator"/>
    <addaction name="actionEqualizer"/>
    <addaction name="separator"/>
    <addaction name="actionEffects"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuPlaylist"/>
   <addaction name="menuOutput"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpenFiles">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open Files</string>
   </property>
   <property name="toolTip">
    <string>Open Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionLogInspector">
   <property name="text">
    <string>Log Inspector ...</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Start playback</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop playback</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="media-skip-backward">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous track</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="media-skip-forward">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next track</string>
   </property>
   <property name="shortcut">
    <string>Alt+Down</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playlist-repeat">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playlist-shuffle">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenURL">
   <property name="icon">
    <iconset theme="folder-remote"/>
   </property>
   <property name="text">
    <string>Open URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAddURL">
   <property name="icon">
    <iconset theme="folder-remote"/>
   </property>
   <property name="text">
    <string>Add URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionNoPlaylistAdvance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No Playlist Advance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionStopAfterThisSong">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop After This Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSongInfo">
   <property name="icon">
    <iconset theme="dialog-information"/>
   </property>
   <property name="text">
    <string>Song Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionJumpToTime">
   <property name="icon">
    <iconset theme="go-jump"/>
   </property>
   <property name="text">
    <string>Jump To Time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionSetRepeatPointA">
   <property name="text">
    <string>Set Repeat Point A</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSetRepeatPointB">
   <property name="text">
    <string>Set Repeat Point B</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionClearRepeatPoints">
   <property name="text">
    <string>Clear Repeat Points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About Audacious</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-system"/>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlayCurrentPlaylist">
   <property name="icon">
    <iconset theme="media-playback-start"/>
   </property>
   <property name="text">
    <string>Play This Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Return</string>
   </property>
  </action>
  <action name="actionPlaylistRefresh">
   <property name="icon">
    <iconset theme="view-refresh"/>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPlaylistRemoveDuplicates">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Remove Duplicates</string>
   </property>
  </action>
  <action name="actionPlaylistRemoveUnavailableFiles">
   <property name="icon">
    <iconset theme="dialog-warning"/>
   </property>
   <property name="text">
    <string>Remove Unavailable Files</string>
   </property>
  </action>
  <action name="actionPlaylistNew">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPlaylistRename">
   <property name="icon">
    <iconset theme="insert-text"/>
   </property>
   <property name="text">
    <string>Rename...</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionPlaylistRemove">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionPlaylistImport">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionPlaylistExport">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionPlaylistManager">
   <property name="icon">
    <iconset theme="audio-x-generic"/>
   </property>
   <property name="text">
    <string>Playlist Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQueueManager">
   <property name="text">
    <string>Queue Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionSortTrackNumber">
   <property name="text">
    <string>By Track Number</string>
   </property>
  </action>
  <action name="actionSortTitle">
   <property name="text">
    <string>By Title</string>
   </property>
  </action>
  <action name="actionSortArtist">
   <property name="text">
    <string>By Artist</string>
   </property>
  </action>
  <action name="actionSortAlbum">
   <property name="text">
    <string>By Album</string>
   </property>
  </action>
  <action name="actionSortDate">
   <property name="text">
    <string>By Release Date</string>
   </property>
  </action>
  <action name="actionSortLength">
   <property name="text">
    <string>By Length</string>
   </property>
  </action>
  <action name="actionSortPath">
   <property name="text">
    <string>By File Path</string>
   </property>
  </action>
  <action name="actionSortCustomTitle">
   <property name="text">
    <string>By Custom Title</string>
   </property>
  </action>
  <action name="actionPlaylistReverse">
   <property name="icon">
    <iconset theme="view-sort-descending"/>
   </property>
   <property name="text">
    <string>Reverse Order</string>
   </property>
  </action>
  <action name="actionPlaylistRandomize">
   <property name="text">
    <string>Random Order</string>
   </property>
  </action>
  <action name="actionSortSelectedTrackNumber">
   <property name="text">
    <string>By Track Number</string>
   </property>
  </action>
  <action name="actionSortSelectedTitle">
   <property name="text">
    <string>By Title</string>
   </property>
  </action>
  <action name="actionSortSelectedArtist">
   <property name="text">
    <string>By Artist</string>
   </property>
  </action>
  <action name="actionSortSelectedAlbum">
   <property name="text">
    <string>By Album</string>
   </property>
  </action>
  <action name="actionSortSelectedDate">
   <property name="text">
    <string>By Release Date</string>
   </property>
  </action>
  <action name="actionSortSelectedLength">
   <property name="text">
    <string>By Length</string>
   </property>
  </action>
  <action name="actionSortSelectedPath">
   <property name="text">
    <string>By File Path</string>
   </property>
  </action>
  <action name="actionSortSelectedCustomTitle">
   <property name="text">
    <string>By Custom Title</string>
   </property>
  </action>
  <action name="actionSelectedReverse">
   <property name="icon">
    <iconset theme="view-sort-descending"/>
   </property>
   <property name="text">
    <string>Reverse Order</string>
   </property>
  </action>
  <action name="actionSelectedRandomize">
   <property name="text">
    <string>Random Order</string>
   </property>
  </action>
  <action name="actionVolumeUp">
   <property name="icon">
    <iconset theme="audio-volume-high"/>
   </property>
   <property name="text">
    <string>Volume Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionVolumeDown">
   <property name="icon">
    <iconset theme="audio-volume-low"/>
   </property>
   <property name="text">
    <string>Volume Down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionEqualizer">
   <property name="icon">
    <iconset theme="multimedia-volume-control"/>
   </property>
   <property name="text">
    <string>Equalizer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionEffects">
   <property name="text">
    <string>Effects ...</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset theme="media-playback-pause"/>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause playback</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
